numpy/_core/tests/test_conversion_utils.py

```diff
@@ -172,9 +172,11 @@ def test_valid(self):
         self._check("no", "NPY_NO_CASTING")
         self._check("equiv", "NPY_EQUIV_CASTING")
         self._check("safe", "NPY_SAFE_CASTING")
-        self._check("same_kind", "NPY_SAME_KIND_CASTING")
```

Comment:
why removed? (maybe leave comment if obvious, since it was not obvious to me...)

Reply:
it was a mistake, good catch. Reverted